.claude/commands/xata-create.yaml

```
@@ -0,0 +1,167 @@
+      result = await xata.db[table].create(
+        recordData,
+        returnFields ? { columns: returnFields.split(',') } : {}
+      );
+    }
+    
+    return result;
+
+output_format:
+  template: |
+    ✅ RECORD CREATED
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)